src/components/Loader/Loader.tsx

```diff
@@ -1,8 +1,18 @@
 import React from 'react';
 import './Loader.scss';
 
-export const Loader: React.FC = () => (
-  <div className="Loader" data-cy="loader">
-    <div className="Loader__content" />
-  </div>
-);
+type Props = {
+  isLoadEnd: boolean,
+};
+
+export const Loader: React.FC<Props> = ({ isLoadEnd }) => {
+  if (!isLoadEnd) {
+    return (
+      <div className="Loader" data-cy="loader">
+        <div className="Loader__content" />
+      </div>
+    );
+  }
+
+  return <></>;
+};
```

Comment:
You already checked it, in other files. I think you shouldn't do if statement here